text/0000-cli-ui.md

```diff
@@ -0,0 +1,96 @@
+- Feature Name: cli-ui-improvements
+- Start Date: 2019-02-25
+- RFC PR: (leave this empty)
+- Notion Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Transition from [Docopt] to [StructOpt], for an easier path to nice user experience (and better developer experience when working on) the command line interface.
```

Comment:
Are completions the only user-visible change? Is there an implication for color? You mention it later but it's not clear to me if DocOpt doesn't support color and StructOpt does?

Reply:
Ah, good clarification. Docopt *can* support them, but you have to manually do every bit of it with e.g. `format!` and `ansi_term`; they work out of the box (and are pretty nice!) with StructOpt.